string output_dir
string ros_path_folder
---
bool success_flag

